fit_resonator/samples/scResTest.ipynb

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
Can you push a simplified python script or something less personalized? This Jupyter notebook includes unnecessary info, like where all your pacakges are installed on your computer.

Reply:
My bad! Just resolved that by moving everything into a .py